pkg/operator/controller/ingress/deployment.go

```diff
@@ -64,6 +64,7 @@ func (r *reconciler) ensureRouterDeleted(ci *operatorv1.IngressController) error
 			return err
 		}
 	}
+	log.Info("deleted deployment for ingress", "namespace", ci.Namespace, "name", ci.Name)
```

Comment:
If it's worth logging at v=0, would an event be better? Not sure I have a consistent set of principles to apply for logging nothing vs. logging v=0, v=1, vs. events in some of these cases. cc @Miciah

Reply:
Both `log.Info` and emitting an event for "I deleted your router" seem reasonable to me.  The operator's log level is hard-coded to "Debug", which is I guess like `v=6` or higher (because glog and zap have to be contrary for whatever reason).